lib/jekyll/converters/scss.rb

```diff
@@ -179,18 +186,53 @@ def sass_configs
         )
       end
 
+      def sass_embedded_config(data)
+        {
```

Comment:
This is intentional. `sass-embedded` uses keyword arguments instead an options hash, and it won't tolerant unknown keyword arguments if passed. Wrapping `sass_build_configuration_options` meaning the options with original names before mapping `sassc` option to `sass-embedded` option will be included and cause `sass-embedded` to refuse those options.

I think it's simpler to just not support unsafe override, rather than sanitize unknown keyword arguments.

Other usable options that supported by `sass-embedded` that is not supported here are `:indent_type`, `:indent_width`, `:linefeed`, which are not that important anyways.